workflow-ui/core-android/api/core-android.api

```diff
@@ -39,17 +39,9 @@ public final class com/squareup/workflow/ui/LifecyclesKt {
 
 public final class com/squareup/workflow/ui/ShowRenderingTag {
 	public fun <init> (Ljava/lang/Object;Lcom/squareup/workflow/ui/ViewEnvironment;Lkotlin/jvm/functions/Function2;)V
-	public final fun component1 ()Ljava/lang/Object;
-	public final fun component2 ()Lcom/squareup/workflow/ui/ViewEnvironment;
-	public final fun component3 ()Lkotlin/jvm/functions/Function2;
-	public final fun copy (Ljava/lang/Object;Lcom/squareup/workflow/ui/ViewEnvironment;Lkotlin/jvm/functions/Function2;)Lcom/squareup/workflow/ui/ShowRenderingTag;
-	public static synthetic fun copy$default (Lcom/squareup/workflow/ui/ShowRenderingTag;Ljava/lang/Object;Lcom/squareup/workflow/ui/ViewEnvironment;Lkotlin/jvm/functions/Function2;ILjava/lang/Object;)Lcom/squareup/workflow/ui/ShowRenderingTag;
-	public fun equals (Ljava/lang/Object;)Z
```

Comment:
Do we not need equals and hashcode for the tag? ToString seemes like it would still be useful.